sklearn/tests/test_naive_bayes.py

```diff
@@ -154,6 +155,31 @@ def test_gnb_pfit_wrong_nb_features():
     assert_raises(ValueError, clf.partial_fit, np.hstack((X, X)), y)
 
 
+def test_gnb_extra_classes():
```

Comment:
new test defined to check if adding new labels at end or started just adds zero-value columns and doesn't change the predictions made earlier.